app/views/entitlements/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<%- @page_title = "#{@entitlement.rank} Entitlements" -%>
```

Comment:
@tpitale Since (for now) `@entitlement` is only used by the `show` action, I'll go with your latter suggestion.